suite-native/atoms/src/Input/Input.tsx

```diff
@@ -78,14 +78,18 @@ const inputStyle = prepareNativeStyle(utils => ({
 
 const inputLabelStyle = prepareNativeStyle(
     (utils, { isLabelMinimized }: Pick<InputWrapperStyleProps, 'isLabelMinimized'>) => ({
-        ...utils.typography.body,
+        lineHeight: utils.typography.body.lineHeight,
```

Comment:
So what's the reason this is breaking animation?

Reply:
fontSize should be handled by the animation style itself, otherwise, it always blinks with fontSize utils.typography.sth when size is defined in both places.

Reply:
Can you add this somewhere comment to code?

Reply:
https://github.com/trezor/trezor-suite/pull/5700/commits/ecf3d30ce9d712820484b60f8d407eba4c835e20